### DISCUSSION TITLE
select special field from json stringify object in nest js

### DISCUSSION THREAD

**Discussion by ftmmlslhi** [prisma/prisma#23441] [Q&A]
## Problem
Im trying to use prisma findMany method to read from mysql object column. Im find a neseted select solution in doc. but it dosent work for me. at last, I was forced to use $rawquery.  this is my query. 
```js
(`SELECT searchRequest ->> "$.ExpireDateTime" as ExpireDateTime, searchRequest ->> "$.TrainItineraryPricingInfo.PTC_FareBreakdowns" as PTC_FareBreakdowns,searchRequest ->> "$.payload.orgId" as OrgId,searchResponse->>'$.TrainItineraryPricingInfo' as TrainItineraryPricingInfo FROM results
            WHERE sessionId = ? AND combinationId = ? `, [
                uuid.getBinaryUUID(request.TrainItinerary.SessionId),
                request.TrainItinerary.CombinationId
            ]);
```
and this is my nested select:
```js
query: {
          where: {
            sessionId: this.uUIDService.getBinaryUUID(request.TrainItinerary.SessionId),
            combinationId: request.TrainItinerary.CombinationId
          },
          select: {
            searchResponse: {
              select : {
                TrainItinerary : {        
                       SessionId: true
                    }
                  }
                }
              }
          }
```
thanks.

**Comment by ludralph** [comment]
Hi @ftmmlslhi 👋 

Thank you for raising this question.

Can you please share your Schema definition and the structure of your JSON? Knowing the exact nesting of properties within `searchRequest` (like `TrainItineraryPricingInfo` and `payload.orgId`) will help us define appropriate filter paths. It would also be beneficial if you can share the exact output as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
